src/app/modules/routing/app-routing.module.ts

```diff
@@ -30,6 +31,14 @@ const routes: Routes = [
       expectedRoles: [UserRoles.MANAGER, UserRoles.ADMIN]
     }
   },
+  {
+    path: 'self-trips',
+    component: SelfTripsComponent,
+    canActivate: [AuthGuard],
+    data: {
+      expectedRoles: [UserRoles.MANAGER, UserRoles.ADMIN]
```

Comment:
should we use that `MANAGER` variable created on user roles here?

Reply:
Yes